docs/deprecations.md

```diff
@@ -19,11 +19,21 @@ being deprecated.
 
 | Feature Being Deprecated                                                                                                                                                                                                     | Deprecation Announcement                                             | [API Compatibility Policy](https://github.com/tektoncd/pipeline/tree/main/api_compatibility_policy.md) | Earliest Date or Release of Removal |
 |------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------|-------------------------------------|
-| [`PipelineResources` are deprecated.](https://github.com/tektoncd/community/blob/main/teps/0074-deprecate-pipelineresources.md)                                                                                              | [v0.30.0](https://github.com/tektoncd/pipeline/releases/tag/v0.30.0) | Alpha                                                                                                  | Dec 20 2021                         |
+| [PipelineResources are deprecated, along with the components of the API that rely on them.](https://github.com/tektoncd/community/blob/main/teps/0074-deprecate-pipelineresources.md) See [Deprecate `PipelineResources` related features](#deprecate-pipelineresources-related-features) for more info.                                                | [v0.30.0](https://github.com/tektoncd/pipeline/releases/tag/v0.30.0) | Alpha                                                                                                  | Dec 20 2021                         |
```

Comment:
NIT: instead of creating a hyperlink with the entire statement, suggest adding a TEP number with a link:

```
PipelineResources are deprecated, along with the components of the API that rely on them as proposed in [TEP-0074](https://github.com/tektoncd/community/blob/main/teps/0074-deprecate-pipelineresources.md)
```

Reply:
Thanks Priti, that makes it look clearer